src/app/content/types.ts

```diff
@@ -42,7 +42,10 @@ export interface State {
 }
 
 export interface PageReferenceMap extends PageReference {
-  match: string;
+  match: {
+    rapMatch: string | undefined,
```

Comment:
why are you pulling around two different types of matches? can't you make them both fit into one match format?

Reply:
because when` contentLinkHandler.ts` searches for hrefs to resolve, rap links come like `href=archive or whatever/something` and the `normal` links in the content come like `href=contents/blah`

Reply:
so what is the problem?

Reply:
That I need to match both href types to replace them with the solved url, I can make it prettier I guess, but I still need to make 2 matches somewhere

Reply:
why do you need two? can't you match either link format and put the data in a consistent match format?

Reply:
Yeah, will work on it.